### PR TITLE
CORTX-32911: Build & deploy CORTX on Multi-node (EC2 Instances) for community build

### PR DESCRIPTION
**Pull request by mukul-seagate11**
# Problem Statement
- Build & deploy CORTX on Multi-node (EC2 Instances) for community build so that community can test & contribute CORTX stack on AWS

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing
Build passed: https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/mukul-space/job/multi-node-community-auto/29/

Deployment Job: https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/mukul-space/job/multi-node-community-auto/35/console

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: jenkins/deployment/community/cortx-community-deploy.groovy [hunk 1]
```diff
@@ -4,217 +4,177 @@ pipeline {
...
+                    export CORTX_DATA_IMAGE=$HOST1:8080/seagate/cortx-data:2.0.0-0 &&
+                    export CORTX_CONTROL_IMAGE=$HOST1:8080/seagate/cortx-control:2.0.0-0 &&
                     sudo env SOLUTION_CONFIG_TYPE=${SOLUTION_CONFIG_TYPE} env CORTX_SERVER_IMAGE=${CORTX_SERVER_IMAGE} env CORTX_CONTROL_IMAGE=${CORTX_CONTROL_IMAGE} env CORTX_DATA_IMAGE=${CORTX_DATA_IMAGE} env COMMUNITY_USE=${COMMUNITY_USE} ./cortx-deploy.sh --cortx-cluster'
                     popd
             '''
             }
         }
-
-        stage ('IO Sanity') {
+        stage('IO Sanity') {
```
Comment (shailesh-vaidya) [inline jenkins/deployment/community/cortx-community-deploy.groovy:158]: Please update name to `Basic I/O Test`

File: jenkins/deployment/community/cortx-community-deploy.groovy [hunk 2]
```diff
@@ -4,217 +4,177 @@ pipeline {
...
+                sh label: 'Destroying EC2 instances........', script: '''
+                pushd solutions/community-deploy/cloud/AWS
+                    terraform destroy -var-file user.tfvars --auto-approve
                     popd
-                    '''
-                }
-
-           script {
-
-            // Jenkins Summary
```
Comment (shailesh-vaidya) [inline jenkins/deployment/community/cortx-community-deploy.groovy:176]: Any specific reason to remove this codebase

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
We should raise all PR for multinode deployment against https://github.com/Seagate/cortx-re/tree/f-94l-multinode

**Comment by mukul-seagate11** [comment]
Will raise another PR for https://github.com/Seagate/cortx-re/tree/f-94l-multinode branch with address all the comments except "//Jenkins Summary" codebase which was failing

**Pull request closed without merging**